core/controllers/classifier.py

```diff
@@ -79,7 +79,10 @@ def verify_signature(message, vm_id, received_signature):
     if secret is None:
         return False
 
-    generated_signature = generate_signature(secret, message)
+    if message is None:
```

Comment:
@prasanna08 . Since there is technically no message here, why don't we have a standard message? Some message dict with one standard key, value which can additionally be used for validation?